packages/lib/registry.ts

```diff
@@ -45,6 +47,15 @@ class Registry {
 		this.showErrorMessageBoxHandler_(message);
 	}
 
+	public setDispatch(dispatch: Dispatch) {
+		this.dispatch_ = dispatch;
+	}
+
+	private sendDispatch(event: AnyAction) {
+		if (!this.dispatch_) throw new Error('Dispatch not set!');
```

Comment:
Please make it optional, defaults to `(action) => {}`. I'm not keen that we have to add this dispatch thing to the registry now, but if there's no other way let's do it, but it should be optional.